From bunk@stusta.de Sun Nov 19 18:23:59 2006
Date: Mon, 20 Nov 2006 03:23:58 +0100
From: Adrian Bunk <bunk@stusta.de>
To: Ping Cheng <pingc@wacom.com>
Cc: gregkh@suse.de, linux-usb-devel@lists.sourceforge.net, linux-kernel@vger.kernel.org
Subject: USB: make drivers/usb/input/wacom_sys.c:wacom_sys_irq() static
Message-ID: <20061120022357.GJ31879@stusta.de>

This patch makes the needlessly global wacom_sys_irq() static.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Ping Cheng <pingc@wacom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/input/wacom.h     |    1 -
 drivers/usb/input/wacom_sys.c |    2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/usb/input/wacom.h
+++ gregkh-2.6/drivers/usb/input/wacom.h
@@ -110,7 +110,6 @@ struct wacom_combo {
 };
 
 extern int wacom_wac_irq(struct wacom_wac * wacom_wac, void * wcombo);
-extern void wacom_sys_irq(struct urb *urb);
 extern void wacom_report_abs(void *wcombo, unsigned int abs_type, int abs_data);
 extern void wacom_report_rel(void *wcombo, unsigned int rel_type, int rel_data);
 extern void wacom_report_key(void *wcombo, unsigned int key_type, int key_data);
--- gregkh-2.6.orig/drivers/usb/input/wacom_sys.c
+++ gregkh-2.6/drivers/usb/input/wacom_sys.c
@@ -42,7 +42,7 @@ static struct input_dev * get_input_dev(
 	return wcombo->wacom->dev;
 }
 
-void wacom_sys_irq(struct urb *urb)
+static void wacom_sys_irq(struct urb *urb)
 {
 	struct wacom *wacom = urb->context;
 	struct wacom_combo wcombo;
